README.md

```diff
@@ -8,23 +8,22 @@ Below are frequently visited Kubecost documentation pages for both the [Commerci
 
 In this guide, you’ll find everything you need to set up Kubecost for your team.
 
-### Quick Installation
+### Quick installation
 
-Check out our [Installation options](https://guide.kubecost.com/hc/en-us/articles/4407601821207) to start monitoring and managing your spend in minutes.
+Check out our [Installation options](/install.md) to start monitoring and managing your spend in minutes.
 
-### Getting Started
+### Getting started
 
 [Troubleshooting installation](https://guide.kubecost.com/hc/en-us/articles/4407601830679)\
```

Comment:
Could we fix these links so that they no longer user "guide.kubecost.com"? Otherwise, looks good to me!